libs/portals/my-pages/social-insurance-maintenance/src/components/PaymentGroupTable/PaymentGroupTable.css.ts

```diff
@@ -0,0 +1,94 @@
+import { theme, themeUtils } from '@island.is/island-ui/theme'
+import { style } from '@vanilla-extract/css'
+
+export const rowLabelColumnCell = style({
+  borderRight: `1px solid ${theme.border.color.blue200} `,
+  boxShadow: `4px 0px 8px -2px ${theme.border.color.blue200}`,
+  left: 0,
+  overflow: 'hidden',
+  position: 'sticky',
+  zIndex: theme.zIndex.above,
+  paddingLeft: theme.spacing[2],
+  paddingRight: 0,
+
+  ...themeUtils.responsiveStyle({
+    lg: {
+      paddingRight: theme.spacing[0],
+      paddingLeft: theme.spacing[1],
+    },
+  }),
+})
+
+export const rowLabelColumnCellBox = style({
+  width: 'auto',
+  ...themeUtils.responsiveStyle({
+    lg: {
+      width: '246px',
+    },
+  }),
+})
+
+export const hidden = style({
+  visibility: 'hidden',
+  margin: 0,
+  padding: 0,
+  border: 'none',
+})
+
+export const noWrap = style({
+  whiteSpace: 'nowrap',
+})
+
+export const expandedColumnCell = style({
+  borderWidth: 0,
+})
+
+export const reset = style({
+  paddingRight: 0,
+  paddingLeft: 0,
+  paddingTop: 0,
+  paddingBottom: 0,
+})
+
+export const nestedLabelColumnCell = style({
+  borderRight: `1px solid ${theme.border.color.blue200} `,
+  boxShadow: `4px 0px 8px -2px ${theme.border.color.blue200}`,
+  left: 0,
+  overflow: 'hidden',
+  position: 'sticky',
+  zIndex: theme.zIndex.above,
```

Comment:
Could maybe be shared with robLabelColumnCell?

Reply:
yeah true